docs/source/daml/code-snippets/Keys.daml

```diff
@@ -112,44 +134,83 @@ lookupTest = scenario do
   submit obs do
     (cid, l) <- fetchByKey @Keyed sig
     assert (keyedCid == cid)
+  -- Stakeholder can't see without authorization
+  submitMustFail obs do visibleByKey @Keyed sig
+  -- Stakeholder can see with authorization
+  submit obs do
+    b <- exercise sigDelegationCid VisibleKeyed with
+      delegee = obs
+      key = sig
+    assert b
   -- Stakeholder can't lookup without authorization
   submitMustFail obs do lookupByKey @Keyed sig
   -- Stakeholder can lookup with authorization
-  submit obs do 
-    mcid <- exercise sigDelegationCid LookupKeyed with 
+  submit obs do
+    mcid <- exercise sigDelegationCid LookupKeyed with
       delegee = obs
       lookupKey = sig
     assert (mcid == Some keyedCid)
 
-  -- Divulgee can't fetch
+  -- Divulgee _can_ fetch the contract directly
+  submit divulgee do
+    exercise obsDelegationCid UnkeyedFetch with
+        delegee = divulgee
+        cid = keyedCid
+  -- Divulgee can't fetch through the key
   submitMustFail divulgee do fetchByKey @Keyed sig
+  -- Divulgee can't see
+  submitMustFail divulgee do visibleByKey @Keyed sig
+  -- Divulgee can't see with stakeholder authority
+  submitMustFail divulgee do
+    exercise obsDelegationCid VisibleKeyed with
+        delegee = divulgee
+        key = sig
   -- Divulgee can't lookup
   submitMustFail divulgee do lookupByKey @Keyed sig
   -- Divulgee can't lookup with stakeholder authority
-  submitMustFail divulgee do 
+  submitMustFail divulgee do
     exercise obsDelegationCid LookupKeyed with
         delegee = divulgee
         lookupKey = sig
-  -- Divulgee can't do positive lookup with maintainer authority. 
+  -- Divulgee can't do positive lookup withm aintainer authority.
```

Comment:
```suggestion
  -- Divulgee can't do positive lookup with maintainer authority.
```
I realize you didn’t introduce this but we might as well fix it now.

Reply:
Good catch.